.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
+/Cargo.lock
```

Comment:
@MarioRodriguezS please write it as `target/` and `Cargo.lock` is a file so shouldn't ends with slash